docker-compose-dev.yml

```diff
@@ -11,6 +11,9 @@ services:
       - NETWORK=prater
       - _DAPPNODE_CONSENSUS_CLIENT_PRATER=prysm-prater.dnp.dappnode.eth
       - _DAPPNODE_EXECUTION_CLIENT_PRATER_VERSION=goerli-geth.dnp.dappnode.eth
+      - _DAPPNODE_GLOBAL_MEVBOOST_MAINNET=false
```

Comment:
should we consider also holesky one?

Reply:
it is pointless to add holesky because we dont have:
- An oracle in Holesky
- A an address of Smooth's contract in Holesky (This would be the FR of Smooth in holesky).

We will have to update and publish a new web3signer when smooth is deployed in holesky. Id say in 1-2 months

Reply:
gotcha